### DISCUSSION TITLE
Calling for new plugin ideas

### DISCUSSION THREAD

**Discussion by yisiliu** [DimensionDev/Maskbook#2066] [Ideas]
Any idea is welcomed here. As long as you think your proposed plugin is interesting or useful to be integrated into Twitter or Facebook, please share with us and we will fund you to realize it!

**Comment by Doinglifewell** [comment]
Hello, 

I am building NFT market place and Dex on crab network, a canary network of [Darwinia.network](Darwinia.network) which is in the polkadot / kusama ecosystem. 

To give you some insight, Darwinia is like cosmos but for substrate, soon their bidirectional universal token bridge will allow all ERC20, BEP20 and ERC721, BEP721 contracts to cross between connected chains.  

I would like to make Crab maskbook compatible to start with, but other substart-based EVM compatible chains should be able to use it as well with little changes needed.
